src/server/app.js

```diff
@@ -151,6 +151,13 @@ app.use((req, res, next) => {
 	res.locals.alerts = [];
 	req.signUpDisabled = false;
 
+	if (process.env.DEPLOY_ENV === 'test' || process.env.DEPLOY_ENV === 'beta') {
+		res.locals.alerts.push({
+			level: 'danger',
+			message: `You are on a ${process.env.DEPLOY_ENV} site`
```

Comment:
Here it would be useful to spell out the difference between beta and test. something like:

> You are on the beta website, which uses the main database but with a newer version of the code to test new features.

and

> You are on the test website; all changes made here are not synced with the main database and will be overwritten periodically. Bugs and issues with the interface are also more likely.